packages/tooltip/src/components/tooltip/Tooltip.module.css

```
@@ -1,5 +1,14 @@
-.noPadding :global(.tippy-content) {
-  padding: 0;
+.tooltip {
+  display: flex;
+  width: fit-content;
+  align-items: center;
+  border-radius: var(--swui-border-radius);
+  padding: var(--swui-metrics-spacing) calc(var(--swui-metrics-spacing) * 2);
```

Om man ska vara sån 🤷🏻‍♀️ 

```suggestion
  padding: var(--swui-metrics-spacing) calc(var(--swui-metrics-indent) * 2);
```